src/ui/FieldTable/FieldValue.ts

```diff
@@ -216,6 +225,17 @@ export class FieldValue extends Component {
     return element;
   }
 
+  /**
+   * Render the caption with the text defined in the component options.<br/>
+   * Returns a <code>HTMLElement</code> containing the rendered caption as text.
+   */
+  public renderTextCaption(): HTMLElement {
+    let element = $$('span').el;
```

Comment:
You could do it like this : 
` $$('span', {className : 'coveo-field-caption'}, _.escape(this.options.textCaption))`